src/services/fetchers/index.ts

```diff
@@ -15,21 +11,17 @@ import EvmTWAPGasPriceFetcher from './EvmTWAPGasPriceFetcher.js';
 import MetalPriceApiFetcher from './MetalPriceApiFetcher.js';
 import MetalsDevApiPriceFetcher from './MetalsDevApiFetcher.js';
 import GoldApiPriceFetcher from './GoldApiPriceFetcher.js';
-import ByBitSpotFetcher from './ByBitSpotFetcher.js';
-import BinancePriceMultiFetcher from './BinancePriceMultiFetcher.js';
+import ByBitFetcher from './ByBitPriceFetcher.js';
+import BinanceFetcher from './BinancePriceFetcher.js';
 
 export {
-  ByBitSpotFetcher,
-  BinancePriceMultiFetcher,
-  CryptoComparePriceMultiFetcher,
-  CryptoCompareHistoDayFetcher,
-  CryptoCompareHistoHourFetcher,
-  CryptoComparePriceWSFetcher,
+  ByBitFetcher as ByBitSpotFetcher,
```

Comment:
I think we should export new names, and in place where we using them, we should create two instances: new and legacy one, this will aloow for backwards compatibility, also when everyone will be using new versions, we can update feeds file and strat using niw unified names.

Reply:
This is unclear to me (I mean the place where the comment is done). The name of the classes do not affect the code behavior. Do you mean on the `FetcherName` enum?

Reply:
why we using `ByBitFetcher as ByBitSpotFetcher`? if name is `ByBitFetcher`?

Reply:
it should be `ByBitFetcher`, however if we have old name in feeds file, it will now work. so we need to set two names for fetchers that requires backwards compatibility so everything can work when we deploy this to our validators only.